torch/_torch_docs.py

```diff
@@ -7327,6 +7327,53 @@ def merge_dicts(*dicts):
 """.format(**factory_common_args))
 
 
+add_docstr(torch.vandermonde_matrix,
+           """
+vandermonde_matrix(x, n, increasing=True, dtype=None, \
+layout=torch.strided, device=None, requires_grad=False) -> Tensor
+""" + r"""
+Generate a Vandermonde matrix.
+
+The columns of the output matrix are powers of the input vector. The order of the powers is 
```

Comment:
Maybe we can make this clearer. What about something like: 

" The columns of the output matrix are elementwise powers of the input vector: x^(n-1), x^(n-2)...x^0. If increasing is true, the order of the columns is reversed: x^0..."?